components/wpt-results.html

```diff
@@ -146,40 +153,31 @@
         testFileResults.forEach(result => {
           const testFiles = result.testFiles
           const resultsURL = result.resultsURL
+          const encodedResultsURL = btoa(result.resultsURL)
 
           Object.keys(result.testFiles).forEach(testFileName => {
```

Comment:
What is done for each test file here? It looks like the purpose of this is to update `this.specs` so that it contains the test file and so that the aggregate summary results are updated, is that right?

(More comments might be helpful here, but I'm not sure.)